src/libstd/io/cursor.rs

```diff
@@ -289,6 +295,16 @@ impl Write for Cursor<Vec<u8>> {
     fn flush(&mut self) -> io::Result<()> { Ok(()) }
 }
 
+#[stable(feature = "cursor_box_slice", since = "1.4.0")]
```

Comment:
Agreed - I have personally been a bit confused as to the exact semantics of some of the impls here. I believe that docs on the impl itself will not be rendered but docs on methods in the impl will.